### DISCUSSION TITLE
Version 2.3.40 Can it be that salt overwrites custum saved objects in dashbords kibana

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#3651] [Unsupported Versions]
Standalone ISO install 2.3.40
8 VCores
16 Gb VRam
2.5 TB disk space

After importing custum made dashboards and visualisations in kibana, I changed the text visualization "Security Onion - Network Data" on the network dashboard with my custom text visualition where the links to two custum dashboards are added.

After a short period the visualization changed back to the default, is it possible that salt overwrite this? Also changes to the "Advanced Settings" are overwritten.

Regards

Bart

**Comment by weslambert** [comment]
You'll need to copy the visualizations and/or dashboards if you do not want them to be overwritten.

**Reply by sleepingbel** [reply]
Hello Wes,

I can see the custum visualization in the saved objects, this are copy's of the originals. I made these in a previous installation of SO 2.3.30 and exported them. I imported them in the new SO 2.3.40 install.

![Knipsel](https://user-images.githubusercontent.com/29328171/113431379-f441b580-93db-11eb-9f79-abf2afb4f899.PNG)

If I replace the original visualization with the custom visualization and save the dashboard, This was what I did in the version 2.3.30.

![Knipsel](https://user-images.githubusercontent.com/29328171/113431650-5ac6d380-93dc-11eb-9808-b7569308fbb9.PNG)

it will be replaced by the original again after a while.

![Knipsel](https://user-images.githubusercontent.com/29328171/113431740-7c27bf80-93dc-11eb-8030-8aefccb8043c.PNG)

This is also the case with changes I make in "Advanced Settings", I change the "Number of rows" back to default and a few seconds later it is back to 100.

All the custum made visualisations and dashboards are in the saved objects list and can be viewed from the saved objects list.

Regards 

Bart

**Reply by sleepingbel** [reply]
Update 06-04-21
I have now copied and modified the original visualization in the SO 2.3.40 and saved it as Security Onion - Network Data - Custum 2

![Knipsel](https://user-images.githubusercontent.com/29328171/113669149-9818b300-96b3-11eb-8100-b6744aaf2e5e.PNG)

Salt keeps overriding the visualization. 
In SO 2.3.30 you could replace a visualization of an original dashboard by a custom visualization and then save the dashboard. Only when you used the command "so-kibana-config-load" it was overwritten.
Now I have the impression that the command "so-kibana-config-load" is executed through a task or salt.
